### PR TITLE
challenge 5

### PR DESCRIPTION
**Pull request by marijaatwork**
Add logout in existing login.cy.js file and add cart tests on the challenge5  branch.

### CODE REVIEW

File: cypress/e2e/cart.cy.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+describe('challenge 5', () => {
+    beforeEach(() => {
+        cy.visit('https://www.saucedemo.com/')
+        cy.get("[data-test='username']").type('standard_user')
+        cy.get("[data-test='password']").type('secret_sauce')
+        cy.get("[data-test='login-button']").click()
+    })
```
Comment (kzivanovic) [inline cypress/e2e/cart.cy.js]: Login can be defined within commands and that way code looks cleaner :))

File: cypress/e2e/cart.cy.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        it('tc1 error with no data', () => {
+          cy.get("[id='shopping_cart_container']").click().end().get("[data-test='checkout']").click()
```
Comment (kzivanovic) [inline cypress/e2e/cart.cy.js:7]: 🚀

File: cypress/e2e/login.cy.js [hunk 3]
```diff
@@ -18,4 +18,13 @@ describe('Challenge 4 new', () => {
...
+    it('test case logout', () =>{
+        cy.visit('https://www.saucedemo.com/')
+        cy.get("[data-test='username']").type('standard_user')
+        cy.get("[data-test='password']").type('secret_sauce')
+        cy.get("[data-test='login-button']").click()
+        cy.get("[id='react-burger-menu-btn']").click()
+        cy.get("[id='logout_sidebar_link']").click()
+        cy.url().should('eq', 'https://www.saucedemo.com/')
+    })
+})
```
Comment (kzivanovic) [inline cypress/e2e/login.cy.js:30]: For practice, you can add here invalid steps for login, and also you can use .clear()

**Review by kzivanovic — APPROVED** [review]

**Comment by marijaatwork** [comment]
@kzivanovic  can you please check the updated cart.cy.js file

**Review by kzivanovic — APPROVED** [review]

**Pull request merged**